src/RegionInfo.java

```diff
@@ -225,6 +225,46 @@ static RegionInfo fromKeyValue(final KeyValue kv,
     return Arrays.copyOf(region_name, comma);
   }
 
+  /**
+   * Given name of a region, returns its start key
+   * @throws IllegalArgumentException if the name of the region is malformed
+   * @param region_name Full region_name created in the constructor
+   * @return byte Array of the start key
+   */
+  static byte[] startKeyFromRegionName(final byte[] region_name){
```

Comment:
This method is just a utility method, and isn't related to any fields / instances of RegionInfo.
(similar kind of RegionInfo#tableFromRegionName())

Maybe we move out these two methods if we know the better place.